kube-fluentd/gcp/fluent.conf

```diff
@@ -0,0 +1,8 @@
+# This is the root config file, which only includes components of the actual configuration
+
+# Do not collect fluentd's own logs to avoid infinite loops.
+<match fluent.**>
+  type null
+</match>
+
+@include /etc/fluent/config/*.conf
```

Comment:
You should at least configure the google-cloud output plugin here, else users will have to configure it themselves whenever they mount the whole config directory.